assembly/models/extrinsic/transfer.ts

```diff
@@ -0,0 +1,147 @@
+import { BytesReader, Codec, CompactInt } from "as-scale-codec";
+import { Utils } from "../../utils";
+
+/**
+ * @description Class representing an Extrinsic in the Substrate Runtime
```

Comment:
I think that this should be: "Class representing a Transfer extrinsic in the Substrate Runtime"